src/main/java/org/graylog/aws/inputs/cloudtrail/notifications/CloudtrailSNSNotificationParser.java

```diff
@@ -1,42 +1,42 @@
 package org.graylog.aws.inputs.cloudtrail.notifications;
 
 import com.amazonaws.services.sqs.model.Message;
+import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.google.common.collect.Lists;
 import org.graylog.aws.inputs.cloudtrail.json.CloudtrailWriteNotification;
 import org.graylog.aws.json.SQSMessage;
 
 import java.io.IOException;
+import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 
 public class CloudtrailSNSNotificationParser {
-    private final ObjectMapper om;
-
-    public CloudtrailSNSNotificationParser() {
-        om = new ObjectMapper();
-    }
+    private final ObjectMapper objectMapper = new ObjectMapper().disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
```

Comment:
Can we inject a single instance across the project to avoid having different configurations?

Reply:
✅